DISCLAIMER.md

```diff
@@ -0,0 +1,21 @@
+# Disclaimer
+
+Simple Icons asks that users read this disclaimer fully before including an icon in their project.
+
+## Table of contents
+
+* [Copyright & Trademark](#copyright--trademark)
+* [Guidelines](#guidelines)
+* [Removal of Brands](#removal-of-brands)
+
+## Copyright & Trademark
+
+The Simple Icons package contains a number of logos with specific requirements around their use. Where this is the case, we aim to provide access to the proper guidelines through the 'guidelines' property in the NPM package, or through links on our website. We ask that our users read these terms fully, and display logos at their own risk. Simple Icons cannot be held responsible for any legal activity raised by a brand. If the icon includes a (registered) trademark icon we follow the guidelines [in our Contribution Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#icon-guidelines) to decide whether to include the symbol or not.
+
+## Guidelines
+
+While we make every effort to provide guidelines on using an icon, brand style guides frequently change and it is up to users of the project to ensure each icon is used correctly and with permission.
+
+## Removal of Brands
+
+Should a brand wish for their icon to be removed from the package, please submit an issue on the [GitHub repo](https://github.com/simple-icons/simple-icons/issues/new/choose) explaining your affiliation with the company, and reasons for removal. We generally remove icons that no longer [meet our criteria](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon) twice a year in our major releases - but can occasionally make exceptions for immediate removal of brands.
```

Comment:
Should we provide an email address here instead? I also think we should include a section on updating icons & data that are either outdated or incorrect, or where the brand don't agree with our treatment of their icon.

Reply:
I feel like yes - is there a way we can set up an email to go to one / all of the @simple-icons/maintainers?

Reply:
> I also think we should include a section on updating icons & data that are either outdated or incorrect, or where the brand don't agree with our treatment of their icon.

I agree that it might be worthwhile to cover both removals and updates


> Should we provide an email address here instead

We should be able to set up an `@simpleicons.org` email address for this given that we own the domain. I'll try to look into the feasibility of this.

Reply:
Quick update, this is possible, but I haven't tried to set it up yet.

Reply:
We're still working on this but I would say we can merge this before we have an email address, agreed @PeterShaggyNoble?